### PR TITLE
Fix pr2_mannequin_mode_nohead.launch

### PR DESCRIPTION
**Pull request by Affonso-Gui**
We had some old paths in `pr2_mannequin_mode_nohead.launch`, so am fixing them and using `unspawn` instead of `stop` to unify with `pr2_mannequin_mode.launch`.

### CODE REVIEW

**Review by v4hn — APPROVED** [review]
The patch looks good to me and I approve. Thanks for fixing this upstream!

@k-okada CI fails, likely due to the same "obsolete indigo" problem.
Could you please have a look?

**Comment by Affonso-Gui** [comment]
@v4hn CI seems to be on place again. @k-okada Thanks for the fix.

**Comment by v4hn** [comment]
Yes, now the only thing missing to merge this request is a second review by @k-okada :-)

**Comment by Affonso-Gui** [comment]
@k-okada Can we ask for a review?

**Comment by k-okada** [comment]
@Affonso-Gui I included this PR into https://github.com/PR2/pr2_apps/pull/40, and waiting for @v4hn ’s review ;-)

**Pull request merged**
